### DISCUSSION TITLE
Wiki link to sample app broken.

### DISCUSSION THREAD

**Discussion by PrajeetShrestha** [xamarin/XamarinCommunityToolkit#690] [Q&A] [answered]
The link to sample app is broken. Also I can't find enough documentations in the wiki. Where can I view all the features provided by XamarinCommunityToolkit?

**Comment by pictos** [comment]
@PrajeetShrestha we don't have official docs yet, they will be released together with the first stable release. For code info, you can check out our sample app.

**Reply by pictos** [reply]
BTW the link for the samples are [here](https://github.com/xamarin/XamarinCommunityToolkit/tree/main/src)

**Comment by jfversluis** [comment, marked as answer]
As mentioned by @pictos we're working on the Docs. The repo for that can be found here: https://github.com/MicrosoftDocs/xamarin-communitytoolkit

When they're published they will be available under docs.microsoft.com. That should happen soon after releasing our first v1. Thanks!
